circus-train-integration-tests/pom.xml

```
@@ -14,9 +15,21 @@
     <hadoop.version>2.8.1</hadoop.version>
     <!-- override for hadoop-common dependency that is used in this test -->
     <guava.version>17.0</guava.version>
+    <log4j-api.version>2.5</log4j-api.version>
```

```suggestion
    <log4j.version>2.5</log4j.version>
```
(it's not just the api)